src/components/Icons/On.js

```diff
@@ -0,0 +1,38 @@
+export default function On() {
```

Comment:
this is SVG was added to change on/off-function in the header. it is not meant to be in this branch and can be ignored as well.

Reply:
If you are going to remove it, just remove it ;)